default/python/AI/ShipDesignAI.py

```diff
@@ -1581,18 +1581,18 @@ def _combat_rating(self):
         return combat_stats.get_rating(enemy_stats=self.additional_specifications.enemy)
 
 
-class WarShipDesigner(ShipDesigner):
+class MilitaryShipDesigner(ShipDesigner):
     basename = "Warship (do not build me)"
```

Comment:
Seems to me we should probably also change this 'Warship' to something like 'Military ship base'